packages/integrations/image/src/index.ts

```diff
@@ -39,7 +39,7 @@ export interface IntegrationOptions {
 	/**
 	 * Entry point for the @type {HostedImageService} or @type {LocalImageService} to be used.
 	 */
-	serviceEntryPoint?: '@astrojs/image/squoosh' | '@astrojs/image/sharp' | string;
+	serviceEntryPoint?: string;
```

Comment:
I think we'd want to preserve the previous static strings too 🤔

Reply:
`string` will override those static strings in intellisense, so I understand the deletion. I actually _just_ went through an extensive twitter thread on this 😄 

TL;DR: if you want to show suggestions via intellisense, but _still_ allow arbitrary strings, you can [either append `& {}` to the definition or use the uppercase `String` object instead](https://twitter.com/BHolmesDev/status/1670575170027769856):

`serviceEntryPoint?: '@astrojs/image/squoosh' | '@astrojs/image/sharp' | String;`

Reply:
As much as I love working around TypeScript 😄 I'll revert these string changes & move https://typescript-eslint.io/rules/no-redundant-type-constituents/ to the todo pile.